worker/pyproject.toml

```diff
@@ -13,24 +13,34 @@ dependencies = [
     "transcribee-proto @ file:///${PROJECT_ROOT}/../proto",
     "numpy>=1.23.5",
     "librosa>=0.10.0",
-    "pydantic>=1.10.5",
+    "pydantic[dotenv]>=1.10.7",
     "whispercpp @ git+https://github.com/transcribee/whispercpp.git@pajowu/wip",
     "transformers>=4.26.1",
-    "torchaudio>=0.13.1",
-    "torch>=1.13.1",
-    "automerge @ git+https://github.com/transcribee/automerge-py.git@rewrite",
+    "torchaudio==0.11.0",
+    "torch==1.11.0",
+    "automerge @ git+https://github.com/transcribee/automerge-py.git@c87d3b39bb271a3b9efcee7aba059b432cf6c09b",
     "websockets>=10.4",
+    "pyannote-audio>=2.1",
+    "torchvision==0.12.0",
+    "hmmlearn @ git+https://github.com/hmmlearn/hmmlearn@e22127765b0ca70d01976df2a369667756b8445e",             # hmmlearn version which actually compiled 🙃. Needs to be specified here instead of in pdm.resolution.overrides because pdm crashes otherwise 🙃🙃
```

Comment:
oof

Reply:
!!. I spend way to much time debugging pdm for these PRs